src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final Random RANDOM = new Random();
+
+    public Ball getRandomBall() {
+        int randomBallNumber = RANDOM.nextInt(100);
```

Comment:
100 is still a magic number)

Reply:
I need to insert the boundary in the constructor?